_sass/_functions.scss

```diff
@@ -27,80 +31,101 @@ $modules: () !default;
 
 // RANGES
 // We use these functions to define ranges for various things, like media queries.
-@function lower-bound($range){
-  @if length($range) <= 0 {
+@function lower-bound($range) {
+  @if length($range) <=0 {
     @return 0;
   }
-  @return nth($range,1);
+
+  @return nth($range, 1);
 }
 
 @function upper-bound($range) {
   @if length($range) < 2 {
     @return 999999999999;
   }
+
   @return nth($range, 2);
 }
 
+/// TODO: This is bad practice, calculating between units should be done by converting as by the sass development team:
+/// https://github.com/sass/sass/issues/533 (right now this is a work around)
```

Comment:
Added this comment because maintainers of sass do not agree with this function existing, so maybe something is going wrong here.